pages.es/common/git-reflog.md

```diff
@@ -13,4 +13,4 @@
 
 - Muestra solo las últimas 5 entradas en el registro de referencias:
 
-`git reflog -n {{5}}`
+`git reflog {{-n|--dry-run}} {{5}}`
```

Comment:
I don't think this is correct. `git reflog -n 5` works, but not `git reflog --dry-run 5`. According to [the docs](https://git-scm.com/docs/git-reflog) the `--dry-run` option is only available for subcommands like `git reflog expire` and `git reflog delete`. 

For reference I tested this out on Git v.2.46.2

Reply:
I'll create a separate PR for pages/common/git-reflog.md to change the example to use `{{-n|--max-count}}`

Reply:
PR created: https://github.com/tldr-pages/tldr/pull/13837